### ISSUE TITLE
[SHADER_REPLACEMENT_...] FORCE_POINT_SAMPLER only works in a small bubble the camera

### ISSUE THREAD

**Issue by PlasticTangerine** [ac-custom-shaders-patch/acc-extension-config#306]
Video: https://streamable.com/5sfndg
Apologies as streamable's compression makes the issue difficult to notice.  If necessary, I could attempt to find an alternative video host with less compression.  Just in case, here is screenshot with some of the noticeable parts highlighted as an alternative.
![Screenshot_arch_mazda_miata_na8c_1994_s1_ptang_hometown_31-6-122-8-29-44](https://user-images.githubusercontent.com/74761423/182027789-29f23aa2-4847-4aa5-982d-d541e620a75d.png)

There seems to be a 2 step "bubble" around the camera where FORCE_POINT_SAMPLER will work.  The distance is not very large and there is a very obvious point that moves along where textures go from sharp to blurry.  

I'm not sure how much of a "bug" this is as I assume this bubble behavior is probably intentional for performance.  However, the transition between the two states is incredibly distracting to the point where it's better if the feature is turned off.

Tested on 0.1.73 and 0.1.78.

**Comment by Marocco2** [comment]
Are you sure this is not related to anisotropic filtering too low?

**Comment by PlasticTangerine** [comment]
I do not think so.  It is set to 16x, and I went and check it again on 0.1.79 and it looks the same.

If you would like to try to reproduce yourself, here is a sample of the same map with point sampler enabled in config.  Launch in practice and start from the pits.

[pointsampler_test.zip](https://github.com/ac-custom-shaders-patch/acc-extension-config/files/10200575/pointsampler_test.zip)
